spec/core_functions/meta/get_mixin/scope.hrx

```diff
@@ -0,0 +1,94 @@
+<===> stores_local_scope/options.yml
+---
+:todo:
+- sass/libsass#2830
+
+<===> stores_local_scope/input.scss
+@use "sass:meta";
+$add-two-mixin: null;
+
+.scope {
+  @mixin add-two($v) {b: $v + 2}
+
+  // This mixin reference will still refer to this nested `add-two` mixin
+  // even when it goes out of scope.
+  $add-two-mixin: get-mixin(add-two) !global;
+}
+
+a {@include meta.apply($add-two-mixin, 10)}
+
+<===> stores_local_scope/output.css
+a {
+  b: 12;
+}
+
+<===>
+================================================================================
+<===> captures_inner_scope/input.scss
+@use "sass:meta";
+@mixin add-two($v) {@error "Should not be called"}
+.scope1 {
+  @mixin add-two($v) {@error "Should not be called"}
+  .scope2 {
+    @mixin add-two($v) {@error "Should not be called"}
+    .scope3 {
+      @mixin add-two($v) {a: $v + 2}
+
+      // Like a normal mixin call, get-mixin() will always use the
+      // innermost definition of a mixin.
+      @include meta.apply(get-mixin(add-two), 10);
+    }
+  }
+}
+
+<===> captures_inner_scope/output.css
+.scope1 .scope2 .scope3 {
+  a: 12;
+}
+
+<===>
+================================================================================
+<===> scope/doesnt-capture-global/input.scss
```

Comment:
[Prefer underscore-separated paths](https://github.com/sass/sass-spec/blob/main/STYLE_GUIDE.md#prefer-underscore-separated-paths). Also, this isn't really accurate—it does capture the global scope (if you pass the mixin reference to a different module it should still refer to the same `$a` value), it's just that the global variable slot it refers to is mutated after the mixin is defined.